extensions/smallrye-opentracing/deployment/src/test/java/io/quarkus/smallrye/opentracing/deployment/TracingTest.java

```diff
@@ -101,7 +101,9 @@ public void testContextPropagationInFaultTolerance() {
             Response response = RestAssured.when().get("/faultTolerance");
             response.then().statusCode(200);
             Assertions.assertEquals("fallback", response.body().asString());
+            Awaitility.await().until(() -> mockTracer.finishedSpans().size() == 5);
```

Comment:
What is the default timeout? Could we make sure it doesn't wait too much?

Reply:
IIRC default is 10s. There are bunch of places in the codebase with 1min delay https://github.com/quarkusio/quarkus/blob/087b16f4d6368aec91a0fddcb7785de8350f1d7b/devtools/maven/src/test/java/io/quarkus/maven/it/RemoteDevMojoIT.java#L223

Reply:
could you maybe reduce that to 5 seconds and squash everything?

I think that should be sufficient.

Reply:
done, you might want to enable squashing via github UI when merging. It removes the burden from contributors and allows changing messages on merge

Reply:
Well... the issue is that we are using merge commits and GH does not have the options to squash and use merge commits. I personally don't like merge commits (especially because the GH history is barely usable) but that's how it is :).

Thanks for doing it.